src/models/parallelCoordinates.js

```diff
@@ -366,7 +377,7 @@ nv.models.parallelCoordinates = function() {
             }
 
             function dragMove(d) {
-                dragging[d.key] = Math.min(availableWidth, Math.max(0, this.parentNode.__origin__ += d3.event.x));
+                dragging[d.key] = Math.min(width, Math.max(0, this.parentNode.__origin__ += d3.event.x));
```

Comment:
width is only set if they use that option, otherwise it's null, why would you want this change?